src/components/BetaLogo.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const BetaLogo = styled.div`
+  padding: 4px 6px;
+  border: 1px solid #1870f3;
+  border-radius: 100px;
+  font-weight: 700;
+  font-size: 11px;
+  line-height: 12px;
+  color: #1870f3;
+  font-family: 'SF Pro Display', sans-serif;
```

Comment:
Adding font family here is normal for this project?

Reply:
Oh no it doesn't need to be here. https://github.com/sendbird/chat-ai-widget/pull/45/commits/9ea081239dce18de26069dbe7c98d4f2bb77cf0b